src/makeCalculator.js

```diff
@@ -33,11 +33,41 @@
  *
  * calculator.result === -28
  *
- *
- * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(x) {
+      calculator.result += x;
+    },
```

Comment:
Change result in operate and reset methods only 
```suggestion
    add(a, b) {
      return a + b;
    },
```